contracts/pegged/OriginalTokenVaults.sol

```diff
@@ -40,22 +48,25 @@ contract OriginalTokenVaults is ReentrancyGuard, Pauser, VolumeControl, DelayedT
      * @notice Lock original tokens to trigger mint at a remote chain's PeggedTokenBridge
      * @param _token local token address
      * @param _amount locked token amount
-     * @param _mintChainId destination chainId to mint tokens
+     * @param _toChainId destination chainId to mint tokens
+     * @param _toAccount destination account to receive minted tokens
      * @param _nonce user input to guarantee unique depositId
      */
     function deposit(
         address _token,
         uint256 _amount,
-        uint64 _mintChainId,
+        uint64 _toChainId,
```

Comment:
`toChainId` is confusing. Technically the tokens are deposited to the current chain

Reply:
updated